sydent/sydent.py

```diff
@@ -328,6 +366,13 @@ def __init__(self, cfg, reactor=twisted.internet.reactor):
             self.cfg.get("email", "email.always_obfuscate")
         )
 
+        self.default_web_client_location = self.cfg.get(
+            "email", "email.default_web_client_location"
+        )
+        self.username_obfuscate_characters = int(self.cfg.get(
```

Comment:
This option has been deprecated on the dinsic branch (see a few lines above for the fallback). So this is harmless as nothing should use it, but heads up that it's not a new config option that we need to set.

Reply:
Right. I'm going to keep it there so that it doesn't conflict with future merges (since we're planning on also deprecating it on mainline)